config.py

```diff
@@ -37,6 +37,13 @@
         'postfix': ('-' + dyno if dyno else '') + '-' + str(os.getpid()),
         'region': 'eu-west-1'
     },
+    's3-parse-tracking-logs': {
+        'bucket': 'hedy-parse-tracking-logs',
```

Comment:
"parse" means having to do with parsing and executing Hedy programs. I don't think this has to do with tracking of just parsed programs, so I would avoid using the word "parse" here. Call it "s3-tracking-logs" everywhere instead.